pkg/descheduler/strategies/node_affinity.go

```diff
@@ -54,7 +54,7 @@ func RemovePodsViolatingNodeAffinity(ctx context.Context, client clientset.Inter
 	}
 	thresholdPriority, err := utils.GetPriorityFromStrategyParams(ctx, client, strategy.Params)
 	if err != nil {
-		klog.V(1).Infof("failed to get threshold priority from strategy's params: %#v", err)
+		klog.V(1).InfoS("failed to get threshold priority from strategy's params", "err", err)
```

Comment:
```suggestion
		klog.V(1).ErrorS(err, "failed to get threshold priority from strategy's params")
```

Reply:
If anything is an error, the code execution should stop at some point. With info, the code is just reporting something happened though not sufficient for a code to stop executing.